test/test_client.py

```diff
@@ -113,27 +113,26 @@ def test_make_offer(self):
         response = client.make_offer([my_asset], [partner_asset], partner_id, 'TESTOWA OFERTA')
         self.assertIsNotNone(response)
 
-
-    def test_make_offer_url(self):
-        sample_trade_url="https://steamcommunity.com/tradeoffer/new/?partner=314218906&token=sgA4FdNm"  # a sample trade url with escrow time of 15 days cause mobile auth not added
+    def test_make_offer_with_trade_url(self):
+        sample_trade_url="https://steamcommunity.com/tradeoffer/new/?partner=314218906&token=sgA4FdNm"
```

Comment:
it is used only if u are sending an offer to a non friend id of yours . the trade url of friends doesnt have it . it is added in the params of sending the offer . without the token i cant send offer to any of my non friends . 'trade_offer_create_params': '{"trade_offer_access_token":"'+token+'"}  . Go to tradeoffers/privacy on the steamcommuntiy and u can generate a trade url with a token that third party websites use to send offer .